tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -459,39 +465,5 @@ spec:
 			stdOut = helper.CmdShouldFail("odo", "unlink", "EtcdCluster/example")
 			Expect(stdOut).To(ContainSubstring("failed to unlink the service"))
 		})
-
-		It("should fail if we delete a link outside of odo (using oc)", func() {
```

Comment:
Removing this test spec because it's about a scenario where the user does something outside the odo workflow.